src/App.jsx

```diff
@@ -1,10 +1,11 @@
 import React from 'react';
 import 'bootstrap/dist/css/bootstrap.min.css';
-import StartButton from './component/StartButton';
+import Styles from './AppStyle.module.css';
```

Comment:
Rename `AppStyle.module.css` to `App.module.css`.
Also, `Styles` should be `styles`